src/Our.Umbraco.DocTypeGridEditor/Bootstrap.cs

```diff
@@ -42,17 +39,20 @@ protected override void ApplicationStarted(UmbracoApplicationBase umbracoApplica
             {
                 if (e.MessageType == MessageType.RefreshByJson)
                 {
-                    var payload = JsonConvert.DeserializeAnonymousType((string)e.MessageObject, new[] { new { Alias = default(string) } });
+                    var payload = JsonConvert.DeserializeAnonymousType((string) e.MessageObject,
+                        new[] {new {Alias = default(string)}});
                     if (payload != null)
                     {
                         foreach (var item in payload)
                         {
-                            applicationContext.ApplicationCache.RuntimeCache.ClearCacheItem(
-                                string.Concat("Our.Umbraco.DocTypeGridEditor.Helpers.DocTypeGridEditorHelper.GetContentTypesByAlias_", item.Alias));
+                            Current.AppCaches.RuntimeCache.ClearByKey(
+                                string.Concat(
+                                    "Our.Umbraco.DocTypeGridEditor.Helpers.DocTypeGridEditorHelper.GetContentTypesByAlias_",
+                                    item.Alias));
```

Comment:
@skttl should we replace this with `$ - string interpolation`?
```
Current.AppCaches.RuntimeCache.ClearByKey($"Our.Umbraco.DocTypeGridEditor.Helpers.DocTypeGridEditorHelper.GetContentTypesByAlias_{item.Alias}");
```

Reply:
Does it really matter? https://koukia.ca/string-interpolation-vs-string-format-string-concat-and-string-builder-performance-benchmarks-c1dad38032a

Besides, it shouldnøt be showstopper for the PR.

Reply:
@skttl no, it compiles to the same. I just think the newer literal notation is a bit cleaner :)